dynawo/sources/ModelicaCompiler/Scripts_OMC_1_13_2/readerOMC.py

```diff
@@ -1226,6 +1230,44 @@ def assign_variables_indexes(self):
         self.nb_integer_vars = index_integer_double
         self.find_calculated_variables()
 
+    def detect_z_only_used_internally(self):
```

Comment:
Adding a comment to explain the three parts of the algo:
I Filling  a dico with z names
II Searching for impact on other z
III Setting the flag based on II and is_when_test

Reply:
Solved